spec.bs

```diff
@@ -2444,6 +2444,35 @@ algorithms to achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-notification.https.html
 </wpt>
 
+<h3 id=cssom-monkeypatch>CSSOM View</h3>
+
+The [[!CSSOM-VIEW]] specification defines scroll bubbling behavior through the [=scroll a target
+into view=] algorithm. Our specification modifies it to not allow scrolling a target into view if it
+is inside of a fenced frame. This is done because the {{Element/scrollIntoView()}} method is an
+event that can both be invoked from and observed by script, which can open a communication side
+channel between a <{fencedframe}> and its embedder.
+
+<div algorithm=scroll-target-into-view>
+  Modify the [=scroll a target into view=] algorithm to add 2 steps after step 1 that read:
+
+  2. If |target|'s associated {{Document}}'s [=node navigable=]'s [=navigable/traversable
+     navigable=] is a [=fenced navigable container/fenced navigable=], terminate these steps.
+
+  3. If |target|'s associated {{Document}}'s [=Document/ancestor navigables=] [list/contains=] a
```

Comment:
Did you read what "ancestor navigables" does? It crawls "parent" navigables until it reaches null, just like the ["traversable navigable"](https://html.spec.whatwg.org/#nav-traversable) algorithm. So it should not be possible to reach this condition. Do you mean to basically use the "unfenced" ancestor navigables, which this PR might be a reason to introduce to this spec? Or is this just an incoherent step?

Reply:
Resolving since this is no longer in the patch.